mypy/newsemanal/semanal.py

```diff
@@ -1699,7 +1702,9 @@ def process_imported_symbol(self,
             if self.final_iteration:
                 self.report_missing_module_attribute(module_id, id, imported_id, context)
                 return
-            self.record_incomplete_ref()
+            else:
+                # This might become a type.
+                self.mark_incomplete(imported_id, node.node, becomes_typeinfo=True)
```

Comment:
I think this requires an update to the `becomes_typeinfo` docs, currently it says "will", not "might".

Reply:
Good point. I'll update this in a separate PR, since I want to merge this now.